integration/network/network_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"github.com/docker/docker/internal/testutils/networking"
```

Comment:
You need to put this import in the block of imports below. I think that's why the CI is complaining.